SU-HSDO/suhumsci
# STN-554 : Fix Colorful Category Pipe tag and width issues

# READY FOR REVIEW

## Summary
This PR resolves the following bugs on category tags in the Colorful theme:
1. The hover state was missing on linked category tags that spanned a single line. Category tags which span multiple lines had a hover state, but it was missing from the last line of anchor text. Solution: The line height has been adjusted to allow the hover state to show when the category tag is a single line and when it spans multiple lines.
2. Inconsistent width of the category pipe. Solution: Use the `transform` property to resize the `width` of the pipe on categories.

## Need Review By (Date)
9/30/2020

## Urgency
low

## Steps to Test
1. In your CLI, run `npm run test` and confirm all our tests pass.
2. Go to http://sparkbox-sandbox.suhumsci.loc/
3. Set the theme to `Colorful`
4. Find the 4 vertical cards located in the `Spotlights` section
- [x] Hover over the category tags to confirm the hover state of an underline
- [x] Open your browser developer tools. Add text to one of the category tag anchor links so that it is long enough to wrap onto another line. Confirm the hover state of an underline exists underline the entire anchor text.
5. Notice the decorative pipe to the left of the category tag. The decorative pipe should be the same width for all cards. In order to replicate the issue discovered in STN-554, zoom your browser window to 80%. (Is was the only way I could replicate the issue.) 
 - [x] Confirm the width of the category pipes is consistent across all cards.
